src/resources/postcss/base/full/forms/_checkboxes-radios.pcss

```diff
@@ -88,6 +94,12 @@
 		&:focus,
 		&:hover {
 			border: 1px solid var(--form-color-border-active);
+			opacity: var(--opacity-icon-hover);
+
+			+ .tribe-common-form-control-checkbox__label,
+			+ .tribe-common-form-control-radio__label {
+				opacity: var(--opacity-icon-hover);
+			}
```

Comment:
this, when compiled, will create:

```
.tribe-common-form-control-checkbox__input:hover + .tribe-common-form-control-checkbox__label,
.tribe-common-form-control-checkbox__input:hover + .tribe-common-form-control-radio__label,
...
```

the checkbox input will never have a sibling radio label, and vice versa, and all we'd be doing is bloating our CSS. let's move the checkboxes to its own and radios to its own (see below).